packages/web3-eth-accounts/src/account.ts

```diff
@@ -15,7 +15,8 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { TransactionFactory, TypedTransaction } from '@ethereumjs/tx';
+import { TypedTransaction } from '@ethereumjs/tx';
+import defaultImport, * as fullImport from '@ethereumjs/tx';
```

Comment:
While local testing in ESM app and importing web3 ESM build there was problem in importing `TransactionFactory` from `@ethereumjs/tx`, reason with ESM `@ethereumjs/tx` is not exposing named export but exporting under default but with CJS `@ethereumjs/tx` is exporting it with named export. Following fix is applied in eth and accounts package and it imports both default and all ( named + default ) making sure that we will not miss `TransactionFactory` in either esm or cjs.

Reply:
This export problem is fixed in @ethereumjs/tx in 4.x and we can upgrade to that in our 4.x after reviewing breaking changes in another PR.

Reply:
Hmm maybe you should try again, now that types are first in export. Maybe also publish repro repo^^

Reply:
It isnt issue in web3 exporting, but in ethereumjs/tx lib. I tested in another proj as well.